pkg/composition/placement-request.go

```diff
@@ -0,0 +1,87 @@
+package composition
```

Comment:
What does `composition` name stand for?

Reply:
Composition as in Go's composition. Where we compose a different object by embedding "sub objects", on this case only one but I can see this being expanded further as other entities are added to the API.